graphql/schemabuilder/types.go

```diff
@@ -85,12 +88,13 @@ func BatchFilterField(name string, batchFilter interface{}) FieldFuncOption {
 	return fieldFuncTextFilterFields
 }
 
-func BatchFilterFieldWithFallback(name string, batchFilter interface{}, filter interface{}, flag func(context.Context) bool) FieldFuncOption {
+func BatchFilterFieldWithFallback(name string, batchFilter interface{}, filter interface{}, flag func(context.Context) bool, options ...FieldFuncOption) FieldFuncOption {
```

Comment:
Do we still need the fieldFuncOptions?  Can we get rid of them?

Reply:
@wedaly was working on a PR to add a tag to filter funcs that were expensive instead of inferring that from if the context is passed in. Since we also want the ability to add the expensive tag to filter fields, I was going to leave it in. 

(This is the PR for reference https://github.com/samsarahq/thunder/pull/281, it will get re-merged after this PR lands)